lib/internal/mime.js

```diff
@@ -122,13 +122,24 @@ const encode = (value) => {
 };
 
 class MIMEParams {
-  #data = new SafeMap();
+  #data = null;
+  #string = null;
+
+  constructor(str) {
```

Comment:
Accepting argument on the class break the spec.
https://bmeck.github.io/node-proposal-mime-api/#interface-mimeparams

Reply:
Well, it depends. If following the WHATWG standard, there's no concept of `MIMEParams` class. It is just an ordered `Map` which is open to plain object, class or even a `Map` instance.

Reply:
fixed